src/cowboy_clock.erl

```diff
@@ -83,10 +83,12 @@ handle_cast(_Msg, State) ->
 	{noreply, State}.
 
 -spec handle_info(any(), State) -> {noreply, State} when State::#state{}.
-handle_info(update, #state{universaltime=Prev, rfc1123=B1, tref=TRef}) ->
+handle_info(update, #state{universaltime=Prev, rfc1123=B1, tref=TRef0}) ->
+	erlang:cancel_timer(TRef0),
```

Comment:
What will happen with erlang:send_after if I make from my code: whereis(cowboy_clock) ! update ?

It will receive update message, create new timer, not cancel old timer and will get 2 update messages during next second.

This will happen only if I send update message from my part of code, of course.